test/conformance/testdata/conformance.txt

```diff
@@ -206,6 +206,8 @@ test/e2e/node/pre_stop.go: "should call prestop when killing a pod"
 test/e2e/scheduling/predicates.go: "validates resource limits of pods that are allowed to run"
 test/e2e/scheduling/predicates.go: "validates that NodeSelector is respected if not matching"
 test/e2e/scheduling/predicates.go: "validates that NodeSelector is respected if matching"
+test/e2e/scheduling/predicates.go: "validates that there is no conflict between pods with same hostPort but different hostIP and protocol"
+test/e2e/scheduling/predicates.go: "validates that there exists conflict between pods with same hostPort and protocol but one using 0.0.0.0 hostIP"
```

Comment:
:/ we should please not use IPv4 only, Kubernetes should be able to be IPv6 as well. cc @aojea

Reply:
I´m working on it, the test is fine, is passing an empty string, but the function I added to translate IPv4 to IPv6 addresses is not checking for that, creating a non-valid IPv6 address
My fault

Reply:
We should also rephrase the test name IMHO.
Thanks for working on this.